submissions/AndriyM/friends-app/style.css

```diff
@@ -0,0 +1,344 @@
+body {
+	background: #f2f4f5;
+	font-family: Montserrat;
+	padding-bottom: 10px;
+}
+
+[class*="__container"] {
+	max-width: 1270px;
+	padding: 0 15px;
+	margin: 0 auto;
+}
+
+@media (max-width: 1098.98px) {
+	[class*="__container"] {
+		max-width: none;
+		padding: 25px;
+	}
+}
+
+@media (max-width: 767.98px) {
+	[class*="__container"] {
+		max-width: none;
+		padding: 0px;
+	}
+}
+
+.wrapper {
+	min-height: 100%;
+	display: flex;
+	flex-direction: column;
+	overflow: hidden;
+}
+
+.main {
+	flex: 1 1 auto;
+}
+
+.content {
+	display: flex;
+}
+
+.header__logo {
+	cursor: pointer;
+}
+
+.header__title {
+	font-size: 48px;
+	font-weight: 800;
+	text-transform: uppercase;
+	padding: 30px 30px 30px 0px;
+	color: rgb(118 90 225);
+	text-shadow: 3px 3px 8px rgb(118 90 225);
+}
+
+/* -------------------------cards ------------------------ */
+.main__cards {
+	padding-left: 30px;
+	display: flex;
+	flex-wrap: wrap;
+	gap: 10px;
+}
+.card {
+	position: relative;
+	background-color: white;
+	letter-spacing: 0px;
+	border-radius: 10px;
+	cursor: pointer;
+	display: flex;
+	flex: 1 0 0;
+}
+
+.card:hover {
+	background-color: rgb(139, 117, 224, 0.4);
+}
+.card:nth-child(2n) {
+	margin-right: 0px;
+}
+
+.card:nth-child(7n),
+.card:nth-child(8n) {
+	margin-bottom: 0px;
+}

```

Comment:
it looks littke bit weird. Please use `justify-content`, `row-gap` and `column-gap` to avoid such kind of `css`.